jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
`data/models` and `data/datasets` are not configuration of the challenge itself.